.github/workflows/ci.yml

```
@@ -0,0 +1,69 @@
+env:
+  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  POSTGRES_USER: test_user
+  POSTGRES_PASSWORD: test_password
+  POSTGRES_DB: uddhar
+
+jobs:
+  backend-ci:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)